specification/cognitiveservices/data-plane/LUIS/Runtime/v2.0/LUIS-Runtime.json

```diff
@@ -0,0 +1,338 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LUIS Endpoint API",
+    "version": "v2.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "$ref": "../../../Common/ExtendedRegions.json#/parameters/AzureRegion"
+      }
+    ]
+  },
+  "basePath": "/luis/v2.0/apps",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/{appId}": {
+      "get": {
+        "description": "Gets the published endpoint predictions for the given query.",
+        "operationId": "Prediction_GetPredictionsFromEndpointViaGet",
+        "parameters": [
+          {
+            "name": "appId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "Format - guid. The application ID."
+          },
+          {
+            "name": "q",
+            "in": "query",
+            "description": "The query to predict.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "timezoneOffset",
+            "in": "query",
+            "description": "The timezone offset for the location of the request.",
+            "type": "number"
+          },
+          {
+            "name": "verbose",
+            "in": "query",
+            "description": "If true will return all intents instead of just the topscoring intent.",
+            "type": "boolean"
+          },
+          {
+            "name": "spellCheck",
+            "in": "query",
+            "description": "Enable spell checking.",
+            "type": "boolean"
+          },
+          {
+            "name": "staging",
+            "in": "query",
+            "description": "Use staging endpoint.",
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A JSON object containing the predictions.",
+            "schema": {
+              "$ref": "#/definitions/LuisResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Successful Get Predictions From Endpoint Via Get request": {
+            "$ref": "./examples/prediction/SuccessfulGetPredictionsFromEndpointViaGetRequest.json"
+          }
+        }
+      },
+      "post": {
+        "description": "Gets the published endpoint prediction for the given long query.",
+        "operationId": "Prediction_GetPredictionsFromEndpointViaPost",
```

Comment:
Are there any fundamental differences between using the `GET` and `POST` endpoints?

Reply:
ACK. The API allows using GET or POST, but the end-result is the same and the parameters constraints are the same too.
For the SDK generation the POST can be used instead. Do you have the configuration required to disable an operation? (Please take a look at the current C# generation @ https://github.com/southworkscom/azure-sdk-for-net/blob/cognitiveservices-luis-runtime-sdk/src/SDKs/CognitiveServices/dataPlane/Language/LUIS-Runtime/Generated/PredictionExtensions.cs#L50 )